pkg/commands/docker.go

```diff
@@ -29,6 +29,7 @@ type DockerCommand struct {
 	Config                 *config.AppConfig
 	Client                 *client.Client
 	InDockerComposeProject bool
+	ShowExited             bool
```

Comment:
I would call this HideExited so that it defaults to false. There are two reasons for this:
1) I think people generally want to see when they've got exited containers
2) Exited containers still take up space on the disk. If somebody has a lot of exited containers, they should probably to a `docker container prune`

Reply:
Agree with you,  
I'll change it then to be default: `true`